httpmon.cc

```diff
@@ -40,6 +40,10 @@ struct ClientControl {
 	bool open;
 	bool deterministic;
 	bool compressed;
+	bool get;
+	bool post;
+    std::string body;
+    std::string header;
```

Comment:
Would it make sense to have `std::vector<std::string> headers`? This would allow multiple headers to be specified.